src/cement_ai_platform/data/processors/time_validation_drift_detection.py

```
@@ -0,0 +1,171 @@
+                drift_summary['most_affected_features'][feature_name] = 0
+            drift_summary['most_affected_features'][feature_name] += 1
+
+            print(f"    - {feature_name}: ", end="")
+            if drift_info['method'] == 'numerical':
+                print(f"JS divergence = {drift_info['js_divergence']:.4f}, KS p-value = {drift_info['ks_p_value']:.4f}")
+            else:
+                print(f"Chi2 p-value = {drift_info['chi2_p_value']:.4f}")
+    else:
+        print("  No drift detected")
+
+print(f"\n\nDrift Detection Summary:")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)